constantine/eth_verkle_ipa/ipa_verifier.nim

```diff
@@ -30,7 +30,7 @@ func generateChallengesForIPA*(res: var openArray[matchingOrderBigInt(Banderwago
     transcript.pointAppend( asBytes"R", proof.R_vector[i])
     res[i].generateChallengeScalar(transcript,asBytes"x")
 
-func checkIPAProof* (ic: IPASettings, transcript: var CryptoHash, commitment: var EC_P, proof: IPAProof, evalPoint: Fr[Banderwagon], res: Fr[Banderwagon]) : bool = 
+func checkIPAProof* (got: var EC_P, ic: IPASettings, transcript: var CryptoHash, commitment: var EC_P, proof: IPAProof, evalPoint: Fr[Banderwagon], res: Fr[Banderwagon]) : bool = 
```

Comment:
argument order is (context, result, the rest), similar to secp256k1: https://github.com/bitcoin-core/secp256k1/blob/0653a25d50f67c68bd2d196ecc7eddab067d95ef/include/secp256k1.h#L14-L17